keps/sig-node/1977-container-notifier/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: Container notifier
+kep-number: 1977
+authors:
+  - "@xing-yang"
+  - "@yuxiangqian"
+owning-sig: sig-node
+participating-sigs:
+  - sig-storage
+status: implementable
+creation-date: 2020-09-20
+reviewers:
+  - "@thockin"
+  - "@liggit"
+  - "@saad-ali"
+  - "@sjenning"
+approvers:
+  - "@thockin"
+  - "@saad-ali"
+  - "@dchen1107"
+prr-approvers:
+  - johnbelamaric
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.20"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.20"
+  beta: "v1.21"
```

Comment:
I think this is too aggressive given that the amount of work.

Reply:
These milestones need to be updated.